packages/protocol/contracts/governance/BlockchainParams.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../common/Initializable.sol";
+
+/**
+ * @title Contract for storing blockchain parameters that can be set by governance.
+ */
+contract BlockchainParams is Ownable, Initializable {
+
+  string public minimumClientVersion;
```

Comment:
Question: do we want this to be a `string` (like a semver or something like that) or a `uint`? String would be more human readable and I guess we already have version strings inherited from geth. An int will be easier to compare, especially if anyone wants to use this value on-chain. @timmoreton, thoughts?

Reply:
I have a slight preference for a triple of ints that match how geth defines them:

```
VersionMajor = 1        // Major version component of the current release
VersionMinor = 9        // Minor version component of the current release
VersionPatch = 2        // Patch version component of the current release
```

we should be able to read and set them all at once.

it reduces the probability we set the value to something that is unparseable.

Reply:
these are golang `int`s  but I think it's fine to use something unsigned and smaller